### PR TITLE
Docker 1.4.1 & Boot2Docker 1.4.1 Support

### PR DESCRIPTION
**Pull request by dmuso**
Historical versions of Homebrew files were:

    fc4e7d263eaea48885bb9425c8a2303e8fe39e2b Library/Formula/docker.rb
    2360263c3fb1c60d8fc4f1bba393611d062d2d34 Library/Formula/boot2docker.rb

### CODE REVIEW

File: boot2docker141.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+require "formula"
+
+class Boot2docker < Formula
```
Comment (DomT4) [inline boot2docker141.rb]: Need to change the ` class ` name to match the formula name here.

File: boot2docker141.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
+require "formula"
```
Comment (DomT4) [inline boot2docker141.rb]: :scissors: this

File: boot2docker141.rb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+class Boot2docker < Formula
+  homepage "https://github.com/boot2docker/boot2docker-cli"
+  # Boot2docker and docker are generally updated at the same time.
+  # Please update the version of docker too
+  url "https://github.com/boot2docker/boot2docker-cli.git", :tag => "v1.4.1"
+  head "https://github.com/boot2docker/boot2docker-cli.git"
+
+  bottle do
```
Comment (DomT4) [inline boot2docker141.rb]: :scissors: this block

File: docker141.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
+require "formula"
+
+class Docker < Formula
```
Comment (DomT4) [inline docker141.rb]: Need to change the ` class ` name to match the formula name here.

File: docker141.rb [hunk 5]
```diff
@@ -0,0 +1,36 @@
+require "formula"
```
Comment (DomT4) [inline docker141.rb]: :scissors: this

File: docker141.rb [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+class Docker < Formula
+  homepage "https://www.docker.com/"
+  # Boot2docker and docker are generally updated at the same time.
+  # Please update the version of boot2docker too
+  url "https://github.com/docker/docker.git", :tag => "v1.4.1"
+
+  bottle do
```
Comment (DomT4) [inline docker141.rb]: :scissors: this block

File: boot2docker141.rb [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    sha1 "9e23c1d530e7ef94fbac0c02eb02fecb943c8f7b" => :yosemite
+    sha1 "72dc462b5bf76c4c69e35aef14e6d4ffa7bbcb6e" => :mavericks
+    sha1 "82da9c5a4f56d4fe5e4fd08d1a2098cb40842bc5" => :mountain_lion
+  end
+
+  depends_on "docker" => :recommended
```
Comment (DomT4) [inline boot2docker141.rb]: Presumably ` depends_on "docker141" => :recommended ` since the two formulae are essentially siblings?

**Comment by dmuso** [comment]
I've made the changes as requested. Any problems, let me know.

**Comment by DomT4** [comment]
Merged! Thanks for your contribution to Homebrew, We appreciate it! :smiley_cat:. For future reference new formula's commit message should be ` docker 1.4.1 (new formula) ` and so on. Thanks!

**Pull request closed without merging**